cmd/prometheus/main.go

```diff
@@ -39,6 +39,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/common/model"
 	"github.com/prometheus/common/version"
+	promRuntime "github.com/prometheus/prometheus/pkg/runtime"
```

Comment:
So far for multi-word package aliases we have always used under_scores rather than camelCase.